images/metering-ansible-operator/roles/meteringconfig/defaults/main.yml

```diff
@@ -167,6 +170,15 @@ _tls_overrides:
             enabled: true
             createSecret: true
             secretName: "{{ meteringconfig_reporting_operator_presto_client_secret_name }}"
+      authProxy:
+        enabled: true
+
+        cookie:
+          seed: "{{ _meteringconfig_reporting_operator_auth_proxy_cookie_seed }}"
+
+        htpasswd:
```

Comment:
Most likely don't need to configure this data ourselves

Reply:
Yeah, this entire `htpasswd` section can be removed